docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Example file for a Drupal-based setup.
+version: '3'
+services:
+
+  web:
+    image: reload/drupal-apache-fpm
+    ports:
+      - '80'
+    depends_on:
+      - fpm
+    volumes:
+      - './:/var/www/web'
+
+  fpm:
+    image: reload/drupal-php7-fpm:7.1
```

Comment:
Why PHP 7.1? PHP 7.2. is the recommended version to use with Drupal 8.5.

Reply:
Yes, fixed.